prompt2model/utils/parse_json_responses.py

```diff
@@ -70,6 +76,10 @@ def parse_prompt_to_fields(
     chat_api = api_tools.default_api_agent
     if max_api_calls and max_api_calls <= 0:
         raise ValueError("max_api_calls must be > 0.")
+
+    if not max_api_calls:
```

Comment:
I think this behavior needs to be documented. If anything, I think this could be replaced with a default value of 5 in the function definition

Reply:
Document it in the sense just add a comment to it in the function docstring right?